### ISSUE TITLE
col names hardcoded in _generate_counterfactual

### ISSUE THREAD

**Issue by knaaptime** [pysal/segregation#148]
[here](https://github.com/pysal/segregation/blob/3c59c8e221cb97ea3d8acf7d72af4b28f2944f04/segregation/util/util.py#L193) and below the columns are hardcoded with strings but they should be variables

**Comment by renanxcortes** [comment]
> [here](https://github.com/pysal/segregation/blob/3c59c8e221cb97ea3d8acf7d72af4b28f2944f04/segregation/util/util.py#L193) and below the columns are hardcoded with strings but they should be variables

Yep, good catch! Further, I think when we drop these columns in https://github.com/pysal/segregation/blob/3c59c8e221cb97ea3d8acf7d72af4b28f2944f04/segregation/util/util.py#L206, this should be variables.
